src/app/dynamic-form/components/flex-container/flex-container.component.html

```diff
@@ -1,6 +1,9 @@
 <div class="flex-container">
     <h3 *ngIf="data.label">{{data.label}}</h3>
-    <div class="flex-content" [fxLayout]="data.layout || 'column'" fxLayout.xs="column" fxLayoutAlign="center">
-        <ng-template appElHost #controls *ngFor="let c of data.controls"></ng-template>
```

Comment:
`@ViewChildren` (#controls) works only in `AfterViewInit` hook.
To be able move generation of dynamic components inside the `OnInit` used ViewChild (#control), and `viewContainerRef.insert`